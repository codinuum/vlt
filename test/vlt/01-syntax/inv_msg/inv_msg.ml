let () = [%debug_log 0]
