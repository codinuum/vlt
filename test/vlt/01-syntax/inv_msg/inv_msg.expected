File "inv_msg.ml", line 1, characters 21-22:
1 | let () = [%debug_log 0]
                         ^
Error: This expression has type int but an expression was expected of type
         string
